scripts/lint_prowjobs/main.go

```diff
@@ -144,7 +144,7 @@ func ServiceAccountCheck(jc *JobConstants) presubmitCheck {
 
 func MakeTargetCheck(jc *JobConstants) presubmitCheck {
 	return presubmitCheck(func(presubmitConfig config.Presubmit, fileContentsString string) (bool, int, string) {
-		if strings.Contains(presubmitConfig.JobBase.Name, "e2e") || strings.Contains(presubmitConfig.JobBase.Name, "lint") || strings.Contains(presubmitConfig.JobBase.Name, "mocks") || presubmitConfig.JobBase.Name == "eks-anywhere-attribution-files-presubmit" || presubmitConfig.JobBase.Name == "eks-anywhere-cluster-controller-tooling-presubmit" || presubmitConfig.JobBase.Name == "eks-anywhere-release-tooling-presubmit" {
+		if strings.Contains(presubmitConfig.JobBase.Name, "e2e") || strings.Contains(presubmitConfig.JobBase.Name, "lint") || strings.Contains(presubmitConfig.JobBase.Name, "mocks") || presubmitConfig.JobBase.Name == "eks-anywhere-attribution-files-presubmit" || presubmitConfig.JobBase.Name == "eks-anywhere-cluster-controller-tooling-presubmit" || presubmitConfig.JobBase.Name == "eks-anywhere-release-tooling-presubmit" || strings.Contains(presubmitConfig.JobBase.Name, "presubmit") {
```

Comment:
This will skip the make target test for all presubmits, since they all have `presubmit` in the job name.

Reply:
> This will skip the make target test for all presubmits, since they all have `presubmit` in the job name.

OK, I took another stab at it. If that's not acceptable, could you explain more of how this function works? I fear I'm not understanding something about its purpose.